### PR TITLE
Upgrade to latest ember-cli

### PR DESCRIPTION
**Pull request by stefanpenner**

### CODE REVIEW

**Comment by stefanpenner** [comment]
Failures appear related to engines.

**Comment by stefanpenner** [comment]
Oh boy, the failure here is juicy!

**Comment by ef4** [comment]
I think the above commit fixes the peerDeps problem. It's not ready to merge until we get stable releases of fixturify-project and scenario-tester, after https://github.com/stefanpenner/node-fixturify-project/pull/50 is reviewed and released. This is pointing at my own alpha releases just to test it.

**Comment by stefanpenner** [comment]
@ef4 I'm assuming you are looking into the failures:

```
   Error: ENOENT: no such file or directory, copyfile 'D:/a/embroider/embroider/packages/compat/src//add-to-tree.d.ts' -> 'C:\Users\RUNNER~1\AppData\Local\Temp\tmp-1716qQfSHqG6WFKD\node_modules\@embroider\compat\src\add-to-tree.d.ts'
        at Object.copyFileSync (fs.js:1991:3)
        at Project.hardLinkFile (D:\a\embroider\embroider\node_modules\scenario-tester\node_modules\fixturify-project\index.js:260:12)
        at Project.hardLinkContents (D:\a\embroider\embroider\node_modules\scenario-tester\node_modules\fixturify-project\index.js:243:22)
        at Project.hardLinkContents (D:\a\embroider\embroider\node_modules\scenario-tester\node_modules\fixturify-project\index.js:240:22)
        at Project.writeLinkedPackage (D:\a\embroider\embroider\node_modules\scenario-tester\node_modules\fixturify-project\index.js:221:14)
        
```

File: tests/scenarios/package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "jsdom": "^16.2.2",
     "lodash": "^4.17.20",
     "qunit": "^2.16.0",
-    "scenario-tester": "^1.0.2",
+    "scenario-tester": "2.0.0",
```
Comment (stefanpenner) [inline tests/scenarios/package.json:12]: what should we do about this?

File: tests/scenarios/package.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
     "jsdom": "^16.2.2",
     "lodash": "^4.17.20",
     "qunit": "^2.16.0",
-    "scenario-tester": "^1.0.2",
+    "scenario-tester": "^2.0.1-alpha.3",
```
Comment (stefanpenner) [inline tests/scenarios/package.json:12]: I'm assuming we are pending `scenario-tester` release?

**Review by stefanpenner — COMMENTED** [review]
LGTM

**Pull request merged**